src/index.html

```diff
@@ -31,7 +33,169 @@
     </nav>
   </header>
 
-  <main></main>
+  <main>
+    <!-- キービジュアル -->
+    <section class="p-keyvisual">
+      <div class="p-keyvisual__img">
+        <div class="p-keyvisual__img__text">
+          <img class="no-1" src="./img/picture/keyvisual-text01.png" width="342" height="169" alt="2021年 使って良かった転職サイトno.1">
+          <h1 class="keyvisualTitle"><img src="./img/picture/keyvisual-text02.png" width="803" height="57" alt="医療事務求人を探すならギズジョブ"></h1>
```

Comment:
[must]
classの命名がFLOCSSのルールに違反してしまってます！
p-keyvisualブロックの中にあるので、以下のように命名してあげる必要がありそうです！

```suggestion
          <h1 class="p-keyvisual__title"><img src="./img/picture/keyvisual-text02.png" width="803" height="57" alt="医療事務求人を探すならギズジョブ"></h1>
```

Reply:
こちら修正しました！
423d0edc8beb5f16cadfab98910503171d0af0a2